City-of-Helsinki/hel-fi-drupal-grants
# feat: AU-1027: Import webforms with drush command

# [UHF-0000](https://helsinkisolutionoffice.atlassian.net/browse/AU-1027)


Paranna DX:ää tarjoamalla drush komennon webformiwn päivittämiseen.

## What was done


Luo moduuli joka määrittää drush komennon

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/AU-1027-import-webforms`
  * `make fresh` tai ainakin aja drush cim
* Run `make drush-cr`


## How to test

* Run `drush grant-tools:webform-import`
* Check that webforms are up to date with the code.

## Designers review



## Other PRs


* Link to other PR
